src/rng.cpp

```diff
@@ -7,8 +8,12 @@ std::default_random_engine random_generator {std::random_device{}()};
 
 void seed_random_number_generator(char const* seed)
 {
-    random_generator.seed(
-        static_cast<unsigned>(djb2_hash(reinterpret_cast<unsigned char const*>(seed))));
+    if (!seed) {
+        // std::random_device returns the same sequence on older version of GCC on windows.
+        random_generator.seed(time(nullptr) ^ 0xDEADBEEF);
```

Comment:
It was @DavidKeaton -- I would probably have just used time() by itself otherwise. But no worries.